### PR TITLE
Update automated aio script for Antelope release

### PR DESCRIPTION
**Pull request by Alex-Welsh**

### CODE REVIEW

File: etc/kayobe/environments/ci-aio/automated-setup.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ if type dnf; then
     sudo dnf -y install git
 else
     sudo apt update
-    sudo apt -y install gcc git libffi-dev python3-dev python-is-python3
+    sudo apt -y install gcc git libffi-dev python3-dev python-is-python3 python3-venv
```
Comment (markgoddard) [inline etc/kayobe/environments/ci-aio/automated-setup.sh:27]: I think we need it in yoga.

Comment (markgoddard) [reply etc/kayobe/environments/ci-aio/automated-setup.sh:27]: and should we update the docs to match?

Comment (Alex-Welsh) [reply etc/kayobe/environments/ci-aio/automated-setup.sh:27]: Will do. Did you come across anything else during the deployment you want to add?

Comment (markgoddard) [reply etc/kayobe/environments/ci-aio/automated-setup.sh:27]: It didn't actually complete, so possibly. I'm not sure what the last error was.

**Comment by Alex-Welsh** [comment]
I think this needs a bit more investigation than my optimistic 2 minute PR. Closing for now

**Pull request closed without merging**